lib/tdf3/src/client/index.ts

```diff
@@ -395,17 +387,18 @@ export class Client {
       metadata,
     });
 
+    const { keyForEncryption, keyForManifest } = await keyMiddleware();
```

Comment:
I feel like we need a context parameter here? IDK if it isn't needed for virtru-sdk maybe we should leave it out. The idea would be a user could use the same keyMiddleware object across multiple encrypt calls or something? 

I am fine with leaving it blank for now as adding a new thing is easier than changing or removing something